### DISCUSSION TITLE
Show beatmap webpage button is not in lazer / add open in browser option on right click for in-game links to website

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#25620] [Ideas] [answered]
There is a button in stable which takes you to a beatmap's webpage from the song select screen, I think this should be in lazer. For convenience there could be an additional option of right clicking the button to display two options: "open in browser" and "open in client" since the in-game browser lacks a bit of functionality which you might want, such as functions like control F and being able to select text to copy etc. Perhaps this right click option window could also extend to other clickable elements which take you to the website in-game, such as mappers' names.

**Comment by huyenden** [comment]
![image](https://github.com/ppy/osu/assets/128424762/bf4bbefb-c326-4034-9ac4-0985a12255c4)
Is this fast?

**Reply by ghost** [reply]
That's on the map page on the in-game browser. I mean having this on the song select screen to link to the map in the in-game browser since it's hard to navigate to individual maps from mapper's profiles - especially if they have a lot of maps. I didn't notice the view in browser button on the map page, the right click thing is kind of unnecessary then since you could just click onto the map page from song select then click to view in browser (if a button on the song select screen was added).

**Reply by peppy** [reply, marked as answer]
You can click through to the overlay and click it. It won't be coming to song select. The website is no longer intended to be useful, as we've moved these interfaces into the game.

**Reply by ghost** [reply]
I didn't notice that you can right click the map in the beatmap carousel (had to look up what this is called) then click on details to go to the map's page in-game, now I see why this feature isn't really needed.
